packages/devtools_app/lib/src/utils.dart

```diff
@@ -617,7 +617,7 @@ class TimeRange {
 }
 
 String formatDateTime(DateTime time) {
-  return DateFormat('h:mm:ss.S a').format(time);
+  return DateFormat('H:mm:ss.S a').format(time);
```

Comment:
if we are using military time, we can get rid of AM / PM

Reply:
Done.